src/site/stages/build/plugins/update-robots.js

```diff
@@ -11,7 +11,7 @@ function updateRobots(buildOptions) {
 
   return files => {
     // Derive the robots file.
-    const robots = files['robots.txt'];
+    const robots = files['generated/robots.txt'];
```

Comment:
I had webpack move this file over because I wanted to take responsibility away from metalsmith, but it could go either way.

Reply:
https://www.va.gov/robots.txt
Since this has more to do with content than it does applications, I think we can leave this in the Metalsmith build.

Reply:
Good call - it's done.